types/proxy-from-env/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for proxy-from-env 1.0
+// Project: https://github.com/Rob--W/proxy-from-env#readme
+// Definitions by: JasonHK <https://github.com/JasonHK>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Takes an input URL and returns the desired proxy URL. If no proxy is set, an
+ * empty string is returned.
+ * @param url The URL
+ * @returns The URL of the proxy that should handle the request to the given
+ *          URL.
+ */
+export function getProxyForUrl(url: string): string;
```

Comment:
I'm not sure how precise you want to be but the docs say it can also accept what node's url.parse returns.

Reply:
This can always be addressed in a future PR.